great_expectations/expectations/metrics/column_aggregate_metrics/column_nonnull_count.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+from great_expectations.compatibility.pyspark import functions as F
+from great_expectations.compatibility.sqlalchemy import sqlalchemy as sa
+from great_expectations.execution_engine import (
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.expectations.metrics.column_aggregate_metric_provider import (
+    ColumnAggregateMetricProvider,
+    column_aggregate_partial,
+    column_aggregate_value,
+)
+
+
+class ColumnNonNullCount(ColumnAggregateMetricProvider):
+    metric_name = "column.nonnull_count"
+    value_keys = ()
+
+    # TODO - Investigate whether we want to use column_aggregate_partial or another decorator
```

Comment:
You can remove this TODO. It looks like the two decorators that aggregate metrics use are `column_aggregate_value` and `column_aggregate_partial`. The partial decorator should let this metric be computed with other aggregate metrics in the same select.